package.json

```diff
@@ -60,6 +60,6 @@
       "typescript": "^4.2.2"
    },
    "peerDependencies": {
-      "next": "^8.1.1-canary.54 || ^9.0.0 || ^10.0.0"
+      "next": "^8.1.1-canary.54 || ^9.0.0 || ^10.0.0 || ^11.0.0"
```

Comment:
This should be all we need to resolve #59, unless there are conflicts elsewhere.